src/components/screens/SchoolScheduleSelectPicture.js

```diff
@@ -190,11 +191,11 @@ class SchoolScheduleSelectPicture extends React.Component {
 	 */
 	nextScreen = () => {
 		this.setImage(this.state.selected);
-		
-		if (this.props.navigation.state.routeName === 'TutorialSchoolScheduleSelectPicture') {
-			this.props.navigation.navigate('TutorialSchoolScheduleCreation');
-		} else {
-			this.props.navigation.navigate('DashboardSchoolScheduleCreation');
+	
+		if(this.props.navigation.state.routeName === TutorialSchoolScheduleSelectPicture) {
```

Comment:
Add back the spaces before the if/else (we forgot to add it in the coding convention)